.gitignore

```diff
@@ -9,6 +9,7 @@ build.properties
 salesforce.schema
 Referenced Packages
 /.sfdx
+**/.vscode/
```

Comment:
@Julian88Tex  -- what is this ** thing I'm seeing here. I've not seen that before.

.vscode/ should be enough I think

Reply:
or is this just in case .vscode appears in subdirectories as well? (even so, in that case I'd think one * would do it...but I could be wrong.

Reply:
So not positive but I believe it's all directories. That being said I literally copy and pasted from Cumulus.

Reply:
So not positive but I believe it's all directories. That being said I literally copy and pasted from Cumulus.

Reply:
no worries. assuming it work (i'll test locally here but sounds good to me)